src/components/SharedComponents/UploadStatus/UploadStatus.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+
+import INatIcon from "components/INatIcon";
+import { View } from "components/styledComponents";
+import * as React from "react";
+import CircularProgressBase from "react-native-circular-progress-indicator";
+import colors from "styles/tailwindColors";
+
+type Props = {
+  color: string,
+  completeColor: string,
+  progress: number
+}
+
+const UploadStatus = ( { color, completeColor, progress }: Props ): React.Node => (
+  <View className="relative justify-center items-center">
+    {( progress < 0.05 )
+      ? <INatIcon className="m-1" name="status-saved" color={colors.darkGray} size={50} />
```

Comment:
After you added fixed dimensions to the View and the icon is centered in it, no margin needed here I guess.

Reply:
Yes, you are right